pokemongo_bot/cell_workers/follow_path.py

```diff
@@ -99,11 +101,14 @@ def find_closest_point_idx(self, points):
         return return_idx
 
     def work(self):
-        last_lat, last_lng, last_alt = self.bot.api.get_position()
+        last_lat = self.bot.api._position_lat
+        last_lng = self.bot.api._position_lng
+        last_alt = self.bot.api._position_alt
 
         point = self.points[self.ptr]
         lat = float(point['lat'])
         lng = float(point['lng'])
+        alt = float(point['alt'])
 
```

Comment:
if 'alt' in point:
    alt = float(point['alt'])
else:
    alt = uniform(self.bot.config.alt_min, self.bot.config.alt_max)

should keep everything good